packages/flutter/lib/src/material/animated_raster.dart

```diff
@@ -0,0 +1,202 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// TODO(jonahwilliams): this is temporarily a private library until we determine
+// a reasonable public API.
+part of page_transitions_theme;
+
+// Foundational widgets for the rasterized zoom page transition.
+
+/// A delegate that controls what is painted during an animation in the [_AnimatedRaster]
+/// widget.
+@immutable
+abstract class _AnimatedRasterDelegate {
+  /// const constructor so that subclasses can be const.
+  const _AnimatedRasterDelegate();
+
+  /// Whether [paint] will paint anything on the current frame.
```

Comment:
This seems inaccurate. you can totally return false from here if you want to paint directly e.g. because an animation you care about is running...